firstvoices/backend/tasks/import_job_tasks.py

```diff
@@ -38,6 +39,25 @@
     "include_on_kids_site",
     "include_in_games",
     "related_entry",
+    # audio
+    "audio_filename",
+    "audio_title",
+    "audio_description",
+    "audio_acknowledgement",
+    "audio_include_in_kids_site",
+    "audio_include_in_games",
+    # image
+    "img_filename",
+    "img_title",
+    "img_description",
+    "img_acknowledgement",
+    "img_include_in_kids_site",
+    # video
+    "video_filename",
+    "video_title",
+    "video_description",
+    "video_acknowledgement",
+    "video_include_in_kids_site",
```

Comment:
Are the "audio", "image", and "video" headers earlier in this list still valid?

Reply:
Good catch, they're not valid now.